### ISSUE TITLE
Crash on RopeGotoDefinition

### ISSUE THREAD

**Issue by mcepl** [python-rope/ropevim#80]
```
Chyba při zpracování function RopeGotoDefinition[1]..provider#python3#Call:
řádek   18:
Error invoking 'python_execute' on channel 3 (python3-script-host):
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/matej/.local/lib/python3.7/site-packages/ropemode/decorators.py", line 54, in newfunc
    return func(*args, **kwds)
  File "/home/matej/.local/lib/python3.7/site-packages/ropemode/interface.py", line 152, in goto_definition
    self._goto_location(definition[0], definition[1])
  File "/home/matej/.local/lib/python3.7/site-packages/ropemode/interface.py", line 435, in _goto_location
    other=self.env.get('goto_def_newwin'))
  File "/home/matej/.local/lib/python3.7/site-packages/ropevim.py", line 248, in find_file
    if self._samefile(win.buffer.name, filename):
  File "/home/matej/.local/lib/python3.7/site-packages/ropevim.py", line 222, in _samefile
    return os.path.samefile(file1, file2)
  File "/usr/lib64/python3.7/genericpath.py", line 96, in samefile
    s1 = os.stat(f1)
FileNotFoundError: [Errno 2] No such file or directory: ''
```

**Comment by benrudolph** [comment]
i also hit this intermittently

**Comment by eivindjahren** [comment]
This might have been solved by #94 , I am testing this now.

**Comment by eivindjahren** [comment]
@lieryan, unless something completely weird is happeing, I can confirm that this was fixed in #94. You can see the successful test here: https://github.com/python-rope/ropevim/pull/96/files#diff-23a7fa2ce62ec62b0ced838a5baa577583b31efc26e6e950af8e817cad348124 . I will be readying up PR 96 soon, so that the testing can facilitate closing some more of these issues.

**Comment by lieryan** [comment]
Thanks for testing this issue @eivindjahren.
